CHANGES.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 5.0.5
+
+- New config option [muc_mention_autocomplete_filter](https://conversejs.org/docs/html/configuration.html#muc_mention_autocomplete_filter)
+- Fix issue [Add user pictures in mention autocomplete list](https://git.happy-dev.fr/startinblox/components/sib-chat/issues/50).
+- Fix issue [Mention names autocomplete](https://git.happy-dev.fr/startinblox/components/sib-chat/issues/95).
```

Comment:
Please don't link to issues from another project, it's not directly related to core Converse development.

Reply:
> Please don't link to issues from another project, it's not directly related to core Converse development.

I appreciate that, but duplicating the issues on converse is an additional administrative task that would be nice to avoid. Does this mean I should stop linking Pade issues to Converse and vice-versa?

The ability to cross-reference the issues may prove to be very useful when an issue is revisited sometime in the future and everyone has forgotten the linkage especially if the project has public access.

Reply:
I think it's OK to link to other issues where relevant, but it's more about *how* one does it.

Many of the bugs fixed and features implemented for this project where done on behalf of (and paid for) other projects and yet we don't link to their issue trackers. It's not clear to me that we should make an exception here.

It makes it look (to me) like this project is being managed by some other issue tracker that belongs to a different project.

>  Does this mean I should stop linking Pade issues to Converse and vice-versa?

What you do in Pade's changelog doesn't concern this project directly. You're free to link however you see fit. Converse is an upstream dependency of Pade and Startin'Blox. In my mind linking to an upstream dependency is different than linking to a downstream project.

>  duplicating the issues on converse is an additional administrative task that would be nice to avoid

I don't expect you to duplicate the issues here, I don't create an issue for every thing I do.

Reply:
OK. Got it